lib/modules/newCommentCount.js

```diff
@@ -269,47 +269,89 @@ addModule('newCommentCount', (module, moduleID) => {
 		});
 	}
 
-	function updateCommentCountFromMyComment() {
-		updateCommentCount(true);
+	function updateCommentCountFromMyComment(entry) {
+		const user = RESUtils.loggedInUser();
+		// don't update for user comment count, if detected new comment wasn't from the user
+		if (user && user === $(entry).find('.author').text()) {
```

Comment:
You could probably use `RESUtils.thing(entry).getAuthor()` for this.  (might have a slightly different name than getAuthor)